src/GraphQL/Execution/ExecutionStrategy.cs

```diff
@@ -204,8 +204,13 @@ protected virtual void SetSubFieldNodes(ExecutionContext context, ObjectExecutio
         /// <inheritdoc/>
         public virtual Dictionary<string, (GraphQLField field, FieldType fieldType)>? GetSubFields(ExecutionContext context, ExecutionNode node)
         {
+            //IGraphType specificType = node is ObjectExecutionNode parent
```

Comment:
I placed comments here because I'm not sure if it helps but at least I tried to align specificType argument value with one from SetSubFieldNodes